docs/source/references/apollo-engine.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Apollo Engine guide
+description: Account management, data privacy, GDPR compliance, and other information about Apollo Engine
+---
+
+[Apollo Engine](https://engine.apollographql.com/) is our cloud service for schema management and performance metrics monitoring. Its foundation is built on two types of data input from servers: publishing schema introspections and sending traces of request execution. From those two data inputs we can provide rich schema usage insights, schema history management, schema change valiation, query usage insights, and more.
+
+Engine's core schema management features are all availble in an unlimited capacity for free, and always will be. Engine's advanced features, like resolver-level query tracing, longer data retention, and third-party integrations are available with subscriptions to the Apollo Team plan.
+
+More information on pricing and billing can be found [here](https://www.apollographql.com/plans/). You can get started with Engine using any GraphQL server by visiting our setup guide [here](/docs/references/setup-analytics.html).
```

Comment:
Let's remove the setup guide sentence and link to `/docs/references/setup-analytics.html` since it no longer exists. We can add it back in once we have the `Turning on analytics` guide completed.